### PR TITLE
fix(testimonials): Improved alt text accessibility.

### LINKED ISSUE #51033: Make alt text on alumni images more accessible
### Describe the Issue

This is a minor issue, but I think it needs to be addressed. The current alt text for the three alumni images on the  main landing page is just the person's name. This is merely duplicating information that is already provided and doesn't give people who can't see the image the same information as people who can see the image.

### Affected Page

https://www.freecodecamp.org/

### Steps to Reproduce

1. Go to 'https://www.freecodecamp.org/'
2. View alt text for each alumni image with the browser's dev tools inspector

### Expected behavior

Alt text for each image should be updated to be a description of the image. This doesn't need to be extremely detailed. Just hit the basic details. Perhaps we could reach out to each of these people and ask them to write their own alt text?

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by dhruvchandak30**
Improved the alt text of Images of Alumni on Landing Page(Testimonials Section).

Closes #51033

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
Hi there, 

Thanks for creating this pull request.

The linked issue has not been triaged yet, and a solution has not been agreed upon. Once the issue is open for contribution, you are welcome to update this pull request to reflect the issue consensus.

**Comment by DylanDevelops** [comment]
I just added a review to this pull request. However, where would the user go to change the alt text of the study group in South Korea?

If we agree to update it, the new alt text should be:
> Five freeCodeCamp students at a local study group in South Korea working together on a laptop.

I assume this is the file to change:
https://github.com/freeCodeCamp/freeCodeCamp/blob/d23676ad8618a75a14412a2d4bdeebe11c077b68/client/src/components/landing/components/campers-image.tsx

The code says `alt={t('landing.hero-img-description')}`. I am unfamiliar with this syntax, so where does this point to?

**Comment by DylanDevelops** [comment]
Yup there it is:

https://github.com/freeCodeCamp/freeCodeCamp/blob/8e07ca93aa39323d0aef10aa7f3bc89d3d2374a3/client/i18n/locales/english/translations.json#L94

 So for the final bit in the pull request, they should update this line to be:
> Five freeCodeCamp students at a local study group in South Korea working together on a laptop.

**Comment by naomi-lgbt** [comment]
I feel like the alt text should be decoupled from the description. It seems repetitive to have the alt text and the caption.

**Comment by bbsmooth** [comment]
@naomi-lgbt Agreed, we don't need to have duplicate information in both the alt text and the caption. Typically, the alt text just describes what's in the image and the caption provides additional information you can't get from the image.

The alt text in this case could be something like "A group of people, including a white man, a black woman, and an Asian woman, gathered around a laptop."

And then the caption would give the information you can't get from the image, which I think the current caption does a pretty good of.

**Comment by raisedadead** [comment]
Hi @DylanDevelops 

I would recommend giving the OP a couple more days to get back to us. You are welcome to open a PR if we don't hear from them by next week? We can leave a comment if you want.

File: client/src/components/landing/components/testimonials.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const Testimonials = (): JSX.Element => {
         <div className='testimonial-card'>
           <div className='testimonial-card-header'>
             <LazyImage
-              alt='Emma Bostian'
+              alt='Emma Bostain, a young-looking White woman, smiling for the camera in front of green foliage.'
```
[the remark this answers is not in the document]
Comment (dhruvchandak30) [reply client/src/components/landing/components/testimonials.tsx:76]: Sorry, Will resolve it.

**Pull request closed without merging**